docs/source/developer/roadmap.md

```diff
@@ -0,0 +1,31 @@
+# Roadmap
+
+## Now
+
+-   More secure method for linking FlowAuth and FlowMachine
+-   Additional home location aggregate
+-   Work locations aggregate
+-   Benchmarking
+-   Cache management
+-   Audit logs
+-   Support for downloading geographies via API
```

Comment:
Might as well stick this in since I've been working on it?
```suggestion
-   Support for downloading geographies via API
-   Improved development setup & workflow
```